packages/redux/src/bags/actions/factories/setBagPromocodesFactory.ts

```diff
@@ -0,0 +1,52 @@
+import * as actionTypes from '../../actionTypes';
+import {
+  Bag,
+  BagPromoCodesInformation,
+  Config,
+  PutBagPromocodes,
+  PutBagPromocodesData,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import type { Dispatch } from 'redux';
+
+/**
+ * Creates a thunk factory configured with the specified client to set.
+ *
+ * @param patchBagItem - Patch bag item client.
```

Comment:
```suggestion
 * @param putBagPromocodes - Put bag promocodes client.
```

Reply:
Fixed 👍

Reply:
The name of the param is still `patchBagItem`, can you change it, please?

Reply:
You are right! Fixed